src/assets/arm_restraints/mittens/mittens.asset.ts

```diff
@@ -27,9 +27,13 @@ DefineAsset({
 		provides: [
 			'Restraint',
 			'Restraint_arms',
+			'Hand_item',
```

Comment:
Okay, yes, I made a mistake in my feedback!

Both mittens absolutely also need Provides 'Hand_Cover' as well, as you thought and showed me.

Please add it and sorry~